docs/publishers/config.rst

```diff
@@ -167,6 +187,22 @@ loads.
 
      This setting can only be set using :js:func:`window.hypothesisConfig`.
 
+   .. option:: onProfileRequest
+
+     ``function``. A JavaScript function that will be called when the user clicks
+     the user profile (user name) link in the sidebar. No arguments are passed
```

Comment:
All valid points. The alternative approach we had discussed earlier was to enable the publisher to provide URL templates as part of the configuration for their namespace. These URL templates could be returned via the `links` API and then the client would not need to do anything.